tests/samplers_tests/test_samplers.py

```diff
@@ -173,12 +167,10 @@ def test_uniform(
 
 
 @parametrize_sampler
-@pytest.mark.parametrize(
-    "distribution", [LogUniformDistribution(1e-7, 1.0), FloatDistribution(1e-7, 1.0, log=True)]
-)
+@pytest.mark.parametrize("distribution", [FloatDistribution(1e-7, 1.0, log=True)])
 def test_log_uniform(
     sampler_class: Callable[[], BaseSampler],
-    distribution: Union[FloatDistribution, LogUniformDistribution],
+    distribution: FloatDistribution,
 ) -> None:
```

Comment:
It is seemingly possible to unify `test_uniform` and `test_log_uniform`. How do you feel to merge them?
If we keep them separated, I prefer to add more test cases or stop using parameterization. 🙋

Reply:
Great catch! I've gone a bit further and merged cases from `test_uniform`, `test_log_uniform` and `test_discrete_uniform` into one `test_float`. Also, there was a line testing `assert np.all(points < distribution.high)`, which is does not seem correct since `FloatDistribution` can include upper range, so i changed that assertion to weak inequality.